device/samples/javascript/device_through_proxy.js

```diff
@@ -53,18 +56,20 @@ function generateMessage () {
   return message;
 }
 
-function errorCallback (err) {
+function errorHandler (err) {
   console.error(err.message);
 }
 
-function connectCallback () {
+function connectHandler () {
   console.log('Client connected');
   // Create a message and send it to the IoT Hub every two seconds
-  sendInterval = setInterval(() => {
-    const message = generateMessage();
-    console.log('Sending message: ' + message.getData());
-    client.sendEvent(message, printResultFor('send'));
-  }, 2000);
+  if (!!sendInterval) {
```

Comment:
Shouldn't this be a single bang? Same feedback applies to the other samples.
```suggestion
  if (!sendInterval) {
```